imap_processing/lo/l0/compression_tables.py

```diff
@@ -0,0 +1,853 @@
+"""Compression tables for decompompressing Lo L0 data."""
+tof_case_table = {
+    # keys: first 4 bits of a compressed binary
+    # values: case number
+    "0000": 0,
+    "0001": 1,
+    "0010": 2,
+    "0011": 3,
+    "0100": 4,
+    "0101": 5,
+    "0110": 6,
+    "0111": 7,
+    "1000": 8,
+    "1001": 9,
+    "1010": 10,
+    "1011": 11,
+    "1100": 12,
+    "1101": 13,
+}
+
+tof_decoder_table = {
+    # First level of keys are the case numbers
+    0: {
+        1: {
```

Comment:
Here you define `[1, 0]`, then in case 1 you define `[1, 2, ..]`. Just verify that the counting is from 0 here and 1 in the next case.

Reply:
I don't think I'm following. I have the outer dictionary as {0, 1, 2, 3, etc.} then the inner dictionary for 0 as {1, 0}. I'm not following where I define 1, 2, in case 1